virtwho/main.py

```diff
@@ -233,6 +240,42 @@ def _main(executor):
 
     return 0
 
+def produce_status_output(result):
+        output = ''
+        if not executor.options[VW_GLOBAL]['json']:
+            output += ("+-------------------------------------------+\n")
+            output += ("           Configuration Status\n")
+            output += ("+-------------------------------------------+\n")
+            for config, report in result.items():
+                output += f"Configuration Name: {config}\n"
+                if 'message' in report.data['source']:
+                    output += f"Source Status: \033[1;31m{report.data['source']['status_string']}\033[0;0m\n"
```

Comment:
I propose to create some constants like `GREEN= \033[1;31m` and use them, because the readability of the current code is not the best and it will be easier to copy-paste code with constants. ;-)